containers/zammad/setup.sh

```diff
@@ -44,7 +44,7 @@ if [ "$1" = 'builder' ]; then
   bundle install
   sed -e 's#.*adapter: postgresql#  adapter: nulldb#g' -e 's#.*username:.*#  username: postgres#g' -e 's#.*password:.*#  password: \n  host: zammad-postgresql\n#g' < contrib/packager.io/database.yml.pkgr > config/database.yml
   sed -i "/require 'rails\/all'/a require\ 'nulldb'" config/application.rb
-  sed -i 's/.*scheduler_\(err\|out\).log.*//g' script/scheduler.rb
+  sed -i 's/.*scheduler_\(err\|out\).log.*//g' script/background-worker.rb
```

Comment:
I believe this line can be dropped, as the new background-worker.rb does not write own logs any more.